react/features/base/config/reducer.js

```diff
@@ -194,6 +194,17 @@ function _translateLegacyConfig(oldValue: Object) {
         newValue.toolbarButtons = interfaceConfig.TOOLBAR_BUTTONS;
     }
 
+    if (!oldValue.connectionIndicators
+    && typeof interfaceConfig === 'object' && (interfaceConfig.CONNECTION_INDICATOR_DISABLED
+        || interfaceConfig.CONNECTION_INDICATOR_AUTO_HIDE_ENABLED
```

Comment:
I think we need to checck for these differently sine they are boolean. We need to check if the object contains those properties, not their values.